eng/common/scripts/Package-Properties.ps1

```diff
@@ -9,29 +9,32 @@ class PackageProps
     [string]$ServiceDirectory
     [string]$ReadMePath
     [string]$ChangeLogPath
+    [string]$SdkType
     [string]$Group
 
-    PackageProps([string]$name, [string]$version, [string]$directoryPath, [string]$serviceDirectory)
+    PackageProps([string]$name, [string]$version, [string]$directoryPath, [string]$serviceDirectory, [string]$sdkType)
     {
-        $this.Initialize($name, $version, $directoryPath, $serviceDirectory)
+        $this.Initialize($name, $version, $directoryPath, $serviceDirectory, $sdkType)
     }
 
-    PackageProps([string]$name, [string]$version, [string]$directoryPath, [string]$serviceDirectory, [string]$group = "")
+    PackageProps([string]$name, [string]$version, [string]$directoryPath, [string]$serviceDirectory, [string]$sdkType, [string]$group = "")
```

Comment:
This order change will likely break any places we are passing in the group already but not the sdktype yet. What is the plan to get sdktype passed in from all the languages?

Reply:
I am creating individual PRs to set this field from each languages and will merge them as soon as this PR is merged. I would prefer to avoid it by defaulting SDKType but this conflicts with second constructor that is already in place.

Reply:
Upon thinking more about this, this will be an ongoing issue whenever we need to add a new property. One more solution is to use an optional Dictionary parameter that can be used to pass any extra properties to PackageProperties Constructor rather than modifying signature every time. PackageProperties class can set the property value if it is present in dictionary.

Reply:
Dictionary sounds good to me. especially for keeping sdktype and track number together.

Reply:
I had a call with Wes and we have identified a better approach to set properties so it will have backward compatibility.